Core/Core/API/APIGroup.swift

```diff
@@ -18,7 +18,7 @@ import Foundation
 
 // https://canvas.instructure.com/doc/api/groups.html#Group
 public struct APIGroup: Codable, Equatable {
-    let id: ID
+    let id: String
```

Comment:
trying to understand why we didn't use a `String` already.

Reply:
There may be an api somewhere that returns it as a number, if so we'll need to switch back. But I can't make it conform to `Context` with an `id: ID`.